packages/hooks/src/useGetEmailVerificationOTP.ts

```diff
@@ -0,0 +1,24 @@
+import { useVerifyEmail } from '@deriv/api';
+import { useStore } from '@deriv/stores';
+import React from 'react';
+
+type TVerifyEmailPayload = Parameters<ReturnType<typeof useVerifyEmail>['mutate']>[0];
+/** A hook for requesting OTP for Email Verification */
+const useGetEmailVerificationOTP = () => {
```

Comment:
we do have a generic hook useVerifyEmail . cant we reuse the same with specifying the type: 'phone_number_verification'

Reply:
I was thinking of handling all the Email Verification here in this hook 🤔 so we just have to call 1 hook in the component for requesting and verifying users.